java/src/hu/holyoil/skeleton/testcases/Example.java

```diff
@@ -15,7 +15,7 @@
     private BillOfMaterial b = new BillOfMaterial();
     @Override
     public String Name() {
-        return "Settler Drills";
+        return "Example";
```

Comment:
The example class is good and all, and i'm really proud of it, but at this point it should be yeeted out of the source code.